component/org.wso2.carbon.identity.oauth2.dpop/src/main/java/org/wso2/carbon/identity/oauth2/dpop/constant/DPoPConstants.java

```diff
@@ -31,7 +31,6 @@ public class DPoPConstants {
     public static final String DPOP_ACCESS_TOKEN_HASH = "ath";
     public static final String DPOP_JWT_TYPE = "dpop+jwt";
     public static final String DPOP_TOKEN_TYPE = "DPoP";
-    public static final String EXPIRED_DPOP_PROOF = "Expired DPoP Proof";
```

Comment:
Why are we removing it?

Reply:
It is not being used anywhere at the moment.